src/ui/connect-ui/src/FamilyHubs.Referral.Web/Pages/Shared/ProfessionalReferralCacheModel.cs

```diff
@@ -22,9 +22,7 @@ protected ProfessionalReferralCacheModel(
         Errors = ErrorState.Empty;
     }
 
-    //todo: change to private set
-    //todo: remove this and reference Errors directly
-    public bool HasErrors { get; set; }
+    public bool HasErrors => Errors.HasErrors;
```

Comment:
The todo about referencing directly might be valid here, save having an extra property that just references the Errors.HasErrors

Reply:
I read that todo as doing what I've done but happy to remove this variable instead.